Source/Directory.build.props

```diff
@@ -50,7 +50,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="SharpGenTools.Sdk" Version="1.1.1" PrivateAssets="All" />
+    <PackageReference Include="SharpGenTools.Sdk" Version="1.1.2" PrivateAssets="All" />
```

Comment:
I would like that whenever we bump SharpGenTools, we have a diff of all generated files, separated from the existing PR, so that we are sure it doesn't introduce a regression in the generated code. Could you post it in a message in this PR?

Reply:
Done

Reply:
> Done

As I don't see a message in the thread, so you ran a diff and there are no changes at all?

Reply:
Not sure I understand correctly?
I added info on main topic and there are separated commits for all update/change I've done.

Reply:
I'm looking for a diff files of all generated files by SharpGenTools, in generated folder, compare from `1.1.1` to `1.1.2`.

These files today are not in the repository, but as it has been a source of regressions in the past months, I would prefer not to update without a confirmation that it doesn't break SharpDX.

The generated files will likely be added to the repository in the future to track this kind of problem, but for now, we need to have a manual check.

Reply:
I see, maybe @jkoritzinsky can help here.

Reply:
> I see, maybe @jkoritzinsky can help here.

@amerkoleci You can do it directly on your machine, revert the change the switch from 1.1.1 to 1.1.2. Run a build
Save the generated files (or add them temporarily to the git index for example, just remove the entry `Generated*/` from `.gitignore`) and switch back to 1.1.2, re-run the build and compare the changes.

Reply:
I've reverted the change and went back to 1.1.1